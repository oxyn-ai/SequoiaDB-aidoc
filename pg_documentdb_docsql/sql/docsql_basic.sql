CREATE EXTENSION pg_documentdb_docsql;
SELECT documentdb_docsql.version();
