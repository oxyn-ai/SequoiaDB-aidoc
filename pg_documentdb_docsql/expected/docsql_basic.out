-- basic load test
CREATE EXTENSION pg_documentdb_docsql;
SELECT documentdb_docsql.version();
              version              
-----------------------------------
 1.0
(1 row)
